xwiki-rendering-macros/xwiki-rendering-macro-toc/src/main/java/org/xwiki/rendering/macro/toc/TocMacroParameters.java

```diff
@@ -176,4 +202,36 @@ public String getReference()
     {
         return this.reference;
     }
+    
+    /**
+     * @param floating Floating with value left or right
+     */
+    public void setFloating(Floating floating)
+    {
+        this.floating = floating;
+    }
+
+    /**
+     * @return current value of floating
+     */
```

Comment:
Same here, don't forget the `@since 10.0RC1`.

Also you don't seems to be using this new parameter anywhere in the macro.